Imagine/Filter/PostProcessor/MozJpegPostProcessor.php

```diff
@@ -88,8 +106,12 @@ public function processWithConfiguration(BinaryInterface $binary, array $options
         $pb = new ProcessBuilder(array($this->mozjpegBin));
 
         // Places emphasis on DC
-        $pb->add('-quant-table');
-        $pb->add(2);
+        $quantTable = array_key_exists('quant_table', $options) ? $options['quant_table'] : $this->quantTable;
+        if (null !== $quantTable) {
```

Comment:
It should be `null` in the constructor, I set to `2` because right now is set to 2, so there is no BC break. Setting `quant-table` to `null` in filters would remove the use of this parameter (which is what I wanted to do).